ats_deployment_guide.rst

```diff
@@ -0,0 +1,170 @@
+
+- PIlz PSSu 4000 ref 314070 
+- PSSu E F 4DI-T 
+- PSSu E F 4DO 0.5-T 
+	
+The configuration of hardware is part of the project where the code is included, son configuration will be explained in section 4.
+
+
+
+Software deployment
+========================
+Each hardware has different software parts, and some hardware had more than one software part. In next sections each hardware element is explained
+	
+Windows Machine
+-------------------
+		
+In the Windows Machine some simulators and some tools are running. 
+Start installing the Force EtherCAT Variables installer, that will install the LabVIEW runtime needed in many other tools and simulators.
+		
+Requirements
+^^^^^^^^^^^^^^^^^^
+
+Force EtherCAT Variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This tool allows to other simulators writing data to EtherCAT variables using a TCP custom protocol. The value written using this tool we be a forcing value, so any slave value will be overwritten with the written value.
+The source code and more documentation about configuration can be found in https://gitlab.tekniker.es/aut/projects/3151-LSST/hil/forceethercatvars.
+Follow next steps to deploy this software
+			
+1. If the installer is available continue in step 6
+2. Clone the repository in the link above
+3. Open the project ForceEtherCATVars.lvproj
+4. Go to “Build Specifications” and right click in “ForceIOs” to select “Build”
+5. Go to “Build Specifications” and right click in “ForceEtherCatVars Installer” to select “Build”
+6. When finishes compilation open location and copy the “Volume” folder to Windows Machine
+7. Install the tool using the “install.exe”
+8. Run ForceIOs.exe.
+		
+Simulate limits
+^^^^^^^^^^^^^^^
+
+This software allows to simulate the behaviour of some subsystem limits switches. Those limits could be part of safety system or EtherCAT distributed IOs.
+The source code and more documentation about configuration can be found in https://gitlab.tekniker.es/aut/projects/3151-LSST/hil/simulatelimits
+Follow next steps to deploy this software
+
+1. If the installer or executable is available continue in step 
+2. Clone the repository in the link above
+3. Open the project SimulateLimits.lvproj
+4. Go to “Build Specifications” and right click in “SimulateLimits” to select “Build”
+5. When build finishes go to build folder and copy all files and folder 
+6. Paste compilation files to desired destination in Windows Machine
+7. Open the "data" folder and open "GeneralConfiguration.xml" 
+8. Change the first path of the field TCP_senders_configuration_Path to point to ForceECATVars_TCP_SenderConfig.xml file in the same data folder.
+9. Change dim='[X]' to dim='[1]' for "TCP_senders_configuration_Path" and for "LimitsDefinition" tags. We are only using the first configured limit because you need the safety full simulator with PILZ hardware to use other limits, When you get this hardware (perhaps you have one in the submit) we can download code to it and use those other limits.
```

Comment:
```suggestion
9. Change dim='[X]' to dim='[1]' for "TCP_senders_configuration_Path" and for "LimitsDefinition" tags. We are only using the first configured limit because you need the safety full simulator with PILZ hardware to use other limits, When you get this hardware (perhaps you have one on the submit) we can download code to it and use those other limits.
```